packages/cli/generators/app/templates/public/index.html.ejs

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title><%= project.description %></title>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Update to the newest design so we have a favicon (otherwise loading this throws a console error in the browser (if someone was to check)).